assignments/arrays.js

```diff
@@ -63,34 +63,73 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+for(let i = 0; i < inventory.length; i++){
+    console.log(`Car 33 is a ${inventory[32]["car_year"]} ${inventory[32]["car_make"]} ${inventory[32]["car_model"]}`); 
+}
 
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+// The dealer needs the information on the last car in their inventory.  
+//What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+for(let i = 0; i < inventory.length; i++){
+    console.log(`The make and model of the last car in the inventory is a ${inventory[inventory.length - 1].car_make} ${inventory[inventory.length - 1].car_model}`); 
```

Comment:
This is a great way to do this so you can see the process. You don't need to loop since you already know it's the very last item. There is a shorter way to consider:
let lastCar = inventory[inventory.length-1];
console.log(lastCar.car_make, lastCar.car_model)